src/webextension/widgets/input.css

```diff
@@ -28,8 +28,9 @@
 
 .input {
   padding: 0 8px;
-  font-size: 13px;
+  font-size: 15px;
```

Comment:
I know this is what the spec *claims* the font size should be, but the spec is wrong. Per the spec, the inputs should use **Body 20**, which is a `15px` high font. However, if you example the text in the example images in the spec, it's *actually* **Body 10**, which is `13px` high as in our code before this change. In addition, the last I checked, nearly every similar input element in Firefox uses `13px` tall text; since our primary goal in implementing Photon visuals is to look like Firefox, I think `13px` is the better choice.

Let's leave this the way it is for now and raise the issue with the Photon design team. If their intent truly is for inputs to have `15px` tall fonts, I'd like to see that indicated consistently in the specs (or at least an explicit note indicating that their sample images are wrong).

Reply:
I was just following the spec in this instance (as well as the designs). And I have found instances where 15px font size is used as they define:
<img width="512" alt="image" src="https://user-images.githubusercontent.com/5185090/36990261-5fedc1e4-2061-11e8-987f-1e0d13017c7b.png">

Reply:
On Linux at least, the exact font size on that field in CSS is `13.3167px`, which gets hinted in height to `14px` (but in width it's about what you'd expect; measuring it, it's about 3% wider than an actual `13px` font, but 10% narrower than `15px`). It may be that there's some platform-specific weirdness happening here too, but based on the inconsistencies in the spec and the OS-level variations we're seeing, I still lean towards implementing the spec as it's *drawn* (i.e. 13px), at least until we get confirmation from the Photon design team about what they really meant in their spec.

I'll file a followup issue and prioritize getting this resolved with the Photon folks, since we really shouldn't have to deal with inconsistent specifications.

Reply:
@jimporter Ya that's interesting, on Mac it shows at ~15. I can go ahead and revert this to 13px until we get clarification from the Photon team though.

<img width="808" alt="screen shot 2018-03-05 at 12 07 34 pm" src="https://user-images.githubusercontent.com/5185090/36994326-fd93b442-206d-11e8-95ad-60afd68eb00d.png">

Reply:
Oh, even better. The spec uses multiple different font sizes on the same page for various images showing how the fields should look. >:(

Reply:
At this point, I honestly can't tell what's right or wrong here, or if there should be variations for each OS or what. I think I'm ok with leaving this at 15px, but we really need the spec fixed so we don't have these issues again.

Reply:
@jimporter Yea I noticed that as well...wish the styles were defined using code vs. images, as that would make things easier. I'll keep it as 15 then, and Ryan and I can connect with the Photon folks to get this resolved.

Reply:
Filed an issue with Photon here: https://github.com/FirefoxUX/photon/issues/283